gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -132,6 +133,7 @@ public synchronized int read() throws IOException {
       return -1;
     }
     if (numRead != 1) {
+      GoogleCloudStorageEventBus.postOnException(new Exception());
```

Comment:
Let us not create the Exception object. Why not add another method, which does not take the actual exception?